### DISCUSSION TITLE
Support

### DISCUSSION THREAD

**Discussion by ncrause** [LoboEvolution/LoboEvolution#64] [Q&A] [answered]
Do you guys use anything like Flattr to take in donations? Building and maintaining a rendering engine like this must be a massive undertaking, and although people might not be able to help with code, a few $$'s a month from users certainly might sweeten the work you do?

**Comment by JasonTheKitten** [comment]
It is definitely hard work xD
I'm also writing my own rendering engine, so I would know xD

As far as a place to donate, it would typically be on the Sourceforge, but the donate button isn't there
https://sourceforge.net/projects/loboevolution
So I'm going to assume they've not yet set up donations.

**Comment by oswetto** [comment, marked as answer]
Hello @ncrause @JasonTheKitten I've added  possibility to become a sponsor of the project.
https://github.com/sponsors/LoboEvolution

**Reply by ncrause** [reply]
Just need to wait for the 15th, but hopefully the project gets more exposure and it becomes more viable to spend dedicated time on it.
